docs/conf.py

```diff
@@ -198,43 +209,58 @@
 
 github_issues_url = 'https://github.com/sunpy/sunpy/issues/'
 
+def rstjinja(app, docname, source):
+    """
+    Render our pages as a jinja template for fancy templating goodness.
+    """
+    # Make sure we're outputting HTML
+    if app.builder.format != 'html':
+        return
+    src = source[0]
+    if "Current status" in src[:20]:
+        rendered = app.builder.templates.render_string(
+            src, app.config.html_context
+        )
+        source[0] = rendered
+
 # -- Options for the Sphinx gallery -------------------------------------------
 
-if on_rtd and os.environ.get('READTHEDOCS_PROJECT').lower() != 'sunpy':
```

Comment:
I got rid of this because why not. I am sure I will be told soon enough why this is important.

Reply:
It's because SunPy has been granted an extra long build time on RTD so the gallery works, if you are not sunpy then the gallery will cause your build to timeout.

Reply:
But we are sunpy?

Reply:
it's to allow people to build their own forks on RTD for testing.

Reply:
Well isn't that what CircleCi is now for? I am not in favour of having people have to setup RTD to build docs.